### DISCUSSION TITLE
Cant connect to web interface

### DISCUSSION THREAD

**Discussion by redwars24** [Security-Onion-Solutions/securityonion#2329] [Unsupported Versions]
Hello,

I am using SO 2.3.10

I was following the tutorial from the SO youtube page.

I installed it from an ISO image. 

I am using the eval

sudo so-status shows all services are running

I ran sudo so-allow then a then the IP I wanted. But when on that host machine, when going to the https://so-eval 

It does not find it. I tried via the IP I assigned and no luck either. Is there something I missed or did wrong? Is it something I need to fix on Virtual Box? I have the adapters on a nat network.

![image](https://user-images.githubusercontent.com/75950059/102021542-708d8b80-3d4e-11eb-8c7e-15632047c2b3.png)

running the so-test command outputs what the video suggested should populate. 

![image](https://user-images.githubusercontent.com/75950059/102021588-c2ceac80-3d4e-11eb-8fcb-89cf718429e1.png)

Also, is there a reason why I must run sudo before any command? 

Thank you!

**Comment by redwars24** [comment]
I can ping the server with success! Just cant load the log in page :(

**Comment by idolaman** [comment]
Have you tried to use so-allow for allowing access to GUI?

**Reply by redwars24** [reply]
Yes, against the entire subnet and the desired IP. Not to mention, it applied the rule during the install.

**Comment by stijnpieters** [comment]
Does `so-eval` actually resolve to the IP address of your VM?

**Reply by redwars24** [reply]
I apologize if I don't understand what you are asking me. What do you mean resolve?

I can ping the SO server from a Windows server VM with the designated IP I gave it, with all the packets received. 

Though from the SO server, I can ping but nothing talks back though.

**Reply by redwars24** [reply]
I even ran a Nmap scan against the server and I can see it.

**Reply by TOoSmOotH** [reply]
when you ping it are you pinging it by name or address?

**Reply by redwars24** [reply]
> when you ping it are you pinging it by name or address?

Address

**Reply by Therics** [reply]
You may need to add the server address and host name in the hosts file, located in the machine you are accessing the web interface from.

**Reply by jsavary87** [reply]
I am having and have been having this exact issue as well. How do you "add the server address and host name in the host file"? I created the VM however I am trying to access the web interface from my host machine. Am I doing that wrong?

**Reply by redwars24** [reply]
> I am having and have been having this exact issue as well. How do you "add the server address and host name in the host file"? I created the VM however I am trying to access the web interface from my host machine. Am I doing that wrong?

C:\Windows\System32\drivers\etc\hosts you have to open that file on notepad. I haven't tried it myself yet though.

**Reply by redwars24** [reply]
> You may need to add the server address and host name in the hosts file, located in the machine you are accessing the web interface from.

Ill try this! Ill let you know. Thanks!

**Reply by jsavary87** [reply]
Sounds good. Thank you!

**Reply by highball8** [reply]
I was having the same issue as @redwars24 and had been looking everywhere for an answer until I found this thread last night. I took the suggestion from @jsavary87 and added the following to the `/etc/hosts` file of the Kali VM that I was trying to use to connect to the web interface at https://so-eval:

```
# Access Security Onion at https://so-eval
<IP_assigned_for_SO_management>     so-eval
```

However, I needed to reboot all of the virtual machines (Security Onion, the Kali VM I was trying to use to access https://so-eval with, and a pfSense VM I am using to create a LAN and mirror traffic) and perhaps even quit VMware Fusion Pro before I was able to get to the web interface.

Thanks, everybody.

**Comment by AndriiStepovyi** [comment]
Question: When you boot Security Onion Guest, in your command line (CLI) it shows you web access address- what is it there (https://so-eval or https://<and some IP address>)?

If it is IP address- you need to compare it with your Security Onion Manager Interface IP address. If they do not match - you need to do several things:

1. Run:

>sudo so-ip-change
#changes SO Manager web access IP

2. Run

>sudo so-allow
#choose a) Analyst and provide your SO Manager's Interface (same as above for web access) IP.

!!! Important: even if it states that it is already allowed- wait untill it fully finishes the script and shows amounts for services it was added for in total (usually something 80-90). You might even see that despite it was already added- it might add 1 more, and that could be exactly the problem it was.

3. Sometimes even with all above set and after >sudo so-status where all services are ok you might see port 80/443 is closed, which means NGINX is not working and need to be restarted. So for that case RUN:
>sudo so-restart nginx

Also should mention that it takes some time for all services to come up after SO booted, usually around several minutes up to 5.
